src/azure-cli/azure/cli/command_modules/cdn/tests/latest/recordings/test_cdn_custom_domain_https_msft.yaml

```diff
@@ -0,0 +1,2323 @@
+  response:
+    body:
+      string: "{\r\n  \"provisioningState\":\"Succeeded\",\"resourceState\":\"Active\"\
+        ,\"hostName\":\"customdomain000002.cdn-cli-test-4.azfdtest.xyz\",\"validationData\"\
+        :null,\"customHttpsProvisioningState\":\"Enabling\",\"customHttpsProvisioningSubstate\"\
+        :\"SubmittingDomainControlValidationRequest\",\"customHttpsParameters\":{\r\
+        \n    \"certificateSource\":\"Cdn\",\"certificateSourceParameters\":{\r\n\
+        \      \"@odata.type\":\"#Microsoft.Azure.Cdn.Models.CdnCertificateSourceParameters\"\
+        ,\"certificateType\":\"Dedicated\"\r\n    },\"minimumTlsVersion\":\"TLS10\"\
+        ,\"protocolType\":\"ServerNameIndication\"\r\n  }\r\n}"
+    headers:
+      azure-asyncoperation:
+      - https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001/providers/Microsoft.Cdn/operationresults/86e3c70e-ead1-4a6a-bd1a-7b7d29587c42?api-version=2019-06-15-preview
+      cache-control:
+      - no-cache
+      content-length:
+      - '546'
+      content-type:
+      - application/json; odata.metadata=minimal
+      date:
+      - Mon, 27 Apr 2020 21:59:28 GMT
+      expires:
+      - '-1'
+      location:
+      - https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001/providers/Microsoft.Cdn/operationresults/86e3c70e-ead1-4a6a-bd1a-7b7d29587c42/profileresults/profile123/endpointresults/cdn-cli-test-4/customdomainresults/customdomain000002?api-version=2019-06-15-preview
+      odata-version:
+      - '4.0'
+      pragma:
+      - no-cache
+      server:
+      - Microsoft-IIS/8.5
+      strict-transport-security:
+      - max-age=31536000; includeSubDomains
+      x-aspnet-version:
+      - 4.0.30319
+      x-content-type-options:
+      - nosniff
+      x-ms-ratelimit-remaining-subscription-writes:
+      - '1199'
+      x-powered-by:
+      - ASP.NET
+    status:
+      code: 202
+      message: Accepted
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      CommandName:
+      - cdn custom-domain enable-https
+      Connection:
+      - keep-alive
+      ParameterSetName:
+      - -g -n --endpoint-name --profile-name --min-tls-version
+      User-Agent:
+      - python/3.7.7 (Darwin-19.4.0-x86_64-i386-64bit) msrest/0.6.10 msrest_azure/0.6.3
+        azure-mgmt-cdn/4.1.0rc1 Azure-SDK-For-Python AZURECLI/2.4.0
+      accept-language:
+      - en-US
+    method: GET
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.Cdn/profiles/profile123/endpoints/cdn-cli-test-4/customDomains/customdomain000002?api-version=2019-06-15-preview
```

Comment:
Why is this happening in multiple places...

Reply:
This GET request is retrieving the custom domain to verify its status is now correct after the previous POST enableCustomHttps request.